src/errors/runtime/message.js

```diff
@@ -24,6 +24,8 @@ export default {
     testNameIsNotAString:                           'The test name is expected to be a string, but it was {type}.',
     testBodyIsNotAFunction:                         'The test body is expected to be a function, but it was {type}.',
     beforeEachIsNotAFunction:                       `Fixture's "beforeEach" method takes a function, but {type} was passed.`,
+    authCredentialIsNotString:                      `Fixture's "httpAuth" method requires a {credential} as a string value, but the {type} value was passed.`,
```

Comment:
If you use type assertions you will have generic message, like:
`"password" options is expected to be a string, but it was {type}`. Don't produce additional messages, use generic ones.